.pylintrc

```diff
@@ -249,7 +249,7 @@ notes=FIXME,XXX,TODO
 [SIMILARITIES]
 
 # Minimum lines number of a similarity.
-min-similarity-lines=4
+min-similarity-lines=6
```

Comment:
This was increased slightly as tests are prone to duplicate code, even with some refactoring there is likely to still be duplicates. Not really worth extra refactors for the sake of having constant for test data between tests which isn't great for readability. Unfortunately, the check cannot be disabled per line/file either. https://github.com/PyCQA/pylint/issues/214